other/materials_designer/specific_examples/twisted_interface_with_commensurate_lattices_molybdenum_disulfide.ipynb

```diff
@@ -0,0 +1,267 @@
+{
```

Comment:
<p>The filename should be <code>interface_bilayer_twisted_commensurate_lattices_molybdenum_disulfide</code> the title should be human-readable </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>